dbms/src/Storages/Transaction/tests/gtest_region_persister.cpp

```diff
@@ -251,7 +251,7 @@ try
         }
 
         // If we truncate page data file, exception will throw instead of droping last region.
-        auto meta_path = path + "/kvstore/page_1_0/meta"; // First page
+        auto meta_path = path + "/page/kvstore/wal/log_1_0"; // First page
```

Comment:
Can it be replace by `getPath()` or something?
Then if the version changed or the paths changed that will be easy to update `gtest`.